docs/naps/8-telemetry.md

```diff
@@ -0,0 +1,191 @@
+:orphan: 
+  
+ (nap-8)= 
+  
+ # NAP-8 — Telemetry
+  
+ ```{eval-rst} 
+ :Author: Grzegorz Bokota 
+ :Created: <date created on, in yyyy-mm-dd format> 
```

Comment:
@Czaki you should update this, I think it's fine to date it as the day you opened the PR.